src/data/panel.ts

```diff
@@ -0,0 +1,22 @@
+import { HomeAssistant, PanelInfo } from "../types";
+import { DEFAULT_PANEL } from "../common/const";
+
+export const getPanelTitle = (hass: HomeAssistant): string | undefined => {
+  let title: string = "";
+  const panel = Object.values(hass.panels).find(
+    (p: PanelInfo): boolean => p.url_path === hass.panelUrl
+  );
+  if (panel) {
```

Comment:
Can you make this a guard clause instead.

```ts
if (!panel) { return ""; }
```

Reply:
:ok_hand: